sql/views_experimental.sql

```diff
@@ -0,0 +1,28 @@
+-- This file and its contents are licensed under the Apache License 2.0.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-APACHE for a copy of the license.
+
+CREATE VIEW timescaledb_experimental.chunk_replication_status AS
```

Comment:
Doesn't DROP of this view need to go into `reverse-dev.sql`?

Reply:
Wondering if it has to do with the fact that the whole schema is dropped since it is part of the same release?

Reply:
It might make more sense to skip the `CASCADE` here and drop the contents in the correct order instead.

Reply:
> Perhaps that test doesn't cover objects in the experimental schema? @mkindahl

Test makes no such distinction, but experimental schema does not exist in 2.3.0, so there are nothing related to experimental schema to check.

Reply:
> > Perhaps that test doesn't cover objects in the experimental schema? @mkindahl
> 
> Test makes no such distinction, but experimental schema does not exist in 2.3.0, so there are nothing related to experimental schema to check.

But I guess that the test should check that the schema doesn't exist (and as a result contains no objects) after a downgrade. Going forward it should check that the schema has the same objects post downgrade as it had before upgrade.

Reply:
I am not sure I understand what you want to work differently.

The update and dowgrade tests are generic and does not have any notion that the experimental schema should or should not exist. It just verifies that upgrading and then downgrading leads to the same result as the original install. Since the schema doesn't exist in the original version, there will be a match and the test will succeed.